### PR TITLE
[Doc] Documentation for AKO HA 

### PR DESCRIPTION
**Pull request by swathinsankaran**
This PR adds the documentation for High Availability in AKO.

### CODE REVIEW

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — COMMENTED** [review]
@swathinsankaran: It will be good if we try to add Few questions asked during HA presentation as FAQ like how to handle Split brain? Can user manually override HA membership? What mechanism is used for deciding Active and Passive AKO? Etc.

File: docs/ako_ha.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Passive AKO does the following:
+* Polls the lease object in the `avi-system` namespace. 
+* Reads the objects in Kubernetes/OpenShift cluster and populates the cache.
+* Reads the AVI objects configured by Active AKO and builds the cache.
+
+## Steps to run AKO in High Availability
+
+* Change the `replicaCount` in `values.yaml` to two before the helm upgrade.
+* Increase the `spec.replicas` to two in the AKO's `statefulset` present in the namespace where the AKO is running after the helm upgrade.
```
[the remark this answers is not in the document]
Comment (swathinsankaran) [reply docs/ako_ha.md]: you're right. I have removed the second option.

**Comment by swathinsankaran** [comment]
@akshayhavile I have updated the document with the FAQ also.

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — DISMISSED** [review]

**Review by akshayhavile — APPROVED** [review]

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — DISMISSED** [review]

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Review by DixitAakash — APPROVED** [review]
LGTM

**Pull request merged**